### PR TITLE
[SBRP-11] Backend - Added Login Function, [SBRP-11] Backend - login a…

### PR DESCRIPTION
**Pull request by oreoanytime**
…nd staff creation changes

[SBRP-11] Backend - Added Login Function
Login endpoint
Maintain session
Added 'Password' column in database under 'Staff' table

[SBRP-11] Backend - login and staff creation changes Login endpoint:
- added try and exception handling Staff Creation endpoint:
-added password into the parameters

### CODE REVIEW

File: backend/database/skills_based_role_portal.sql [hunk 1]
```diff
@@ -31,16 +31,17 @@ CREATE TABLE IF NOT EXISTS Staff (
     Country VARCHAR(50) NOT NULL,
     Email VARCHAR(50) UNIQUE NOT NULL,
     Access_Rights INT,
+    Password VARCHAR(50) NOT NULL,
     FOREIGN KEY (Access_Rights) REFERENCES AccessRights(Access_Rights_ID)
 );
 
 INSERT INTO Staff (Staff_FName, Staff_LName, Dept, Country, Email, Access_Rights)
```
Comment (L1uM4rt1n) [inline backend/database/skills_based_role_portal.sql]: line 38 missing password parameter -- wrong line highlighted paiseh

**Review by L1uM4rt1n — CHANGES_REQUESTED** [review]
check through sql file again

File: backend/database/skills_based_role_portal.sql (hunk 1 above)
Comment (oreoanytime) [reply backend/database/skills_based_role_portal.sql]: resolved

**Review by L1uM4rt1n — APPROVED** [review]
all reviewed and good now

**Pull request merged**